resources/assets/js/app.js

```diff
@@ -134,30 +133,34 @@ $(document).ready(function(){
       data: data,
       processData: false,
       contentType: false,
-      beforeSend: function(){
-        $("body").css("cursor", "progress");
-        $("div").removeClass("has-error");
-        $(".help-block").remove();
+      beforeSend: function () {
+        $('body').css('cursor', 'progress');
```

Comment:
Just out of curiosity, any reason for the double -> single quote change?  Not sure if there was some intent other than "appease StyleCI"

Reply:
No, It was just a personal preference. I use double quotes for strings where I do interpolation and single quotes for plain strings. But that's on PHP's side. But on javascript they behave the same AFAIK.